PinpointKit/PinpointKit/Sources/BoxAnnotationView.swift

```diff
@@ -16,7 +16,12 @@ public class BoxAnnotationView: AnnotationView {
     /// The corresponding annotation.
     var annotation: BoxAnnotation? {
         didSet {
-            layer.shadowPath = annotation.flatMap(PathForDrawingBoxAnnotation)?.cgPath
+            if let annotation = annotation {
+                layer.shadowPath = self.dynamicType.path(forDrawing: annotation)?.cgPath
```

Comment:
It’s a bit strange that we’re saying `forDrawing` and then passing an annotation, no?